src/Link_TEST.cc

```diff
@@ -302,6 +316,353 @@ TEST(DOMLink, ResolveAutoInertialsWithNoCollisionsInLink)
   EXPECT_NE(nullptr, root.Element());
 }
 
+/////////////////////////////////////////////////
+TEST(DOMLink, ResolveAutoInertialsWithDifferentDensity)
+{
+  std::string sdf = "<?xml version=\"1.0\"?>"
```

Comment:
nit: I think string literals would be great here like https://github.com/gazebosim/sdformat/blob/11205c9059cd3991564b97d525054fcabc40faff/src/Root_TEST.cc#L242, but if I won't block on this.

Reply:
https://github.com/gazebosim/sdformat/pull/1335/commits/b41b48b612db47a30409b80b4d1bb793ab734377